tests/remove_neg_test.py

```diff
@@ -94,16 +94,21 @@ def test_remove_neg(self) -> None:
         with self.subTest("09_test_server_received_rpcs_from_test_client"):
             self.assertSuccessfulRpcs(test_client)
 
-        with self.subTest("10_remove_neg"):
-            self.assertRpcsEventuallyGoToGivenServers(
-                test_client, default_test_servers + same_zone_test_servers
-            )
+        with self.subTest("10_remove_neg_alternate"):
             self.removeServerBackends(
                 server_runner=self.alternate_server_runner
             )
             self.assertRpcsEventuallyGoToGivenServers(
                 test_client, default_test_servers
             )
+            # Bring up backend for next step
+            self.setupServerBackends(server_runner=self.alternate_server_runner)
```

Comment:
Could be a bit confusing to have this in 10_remove subtest. Maybe move it to `with self.subTest("11_add_neg_alternate"):`?